builder/amazon/ebs/builder.go

```diff
@@ -234,6 +234,7 @@ func (b *Builder) Run(ctx context.Context, ui packer.Ui, hook packer.Hook) (pack
 		},
 		&awscommon.StepIamInstanceProfile{
 			IamInstanceProfile:                        b.config.IamInstanceProfile,
+			SkipProfileValidation:                     b.config.SkipProfileValidation,
```

Comment:
add this same configuration to the other amazon builders and the related generated files
[./builder/amazon/ebssurrogate/builder.go](https://github.com/hashicorp/packer/blob/master/builder/amazon/ebssurrogate/builder.go#L258)
[./builder/amazon/ebsvolume/builder.go](https://github.com/hashicorp/packer/blob/master/builder/amazon/ebsvolume/builder.go#L228)
[./builder/amazon/instance/builder.go](https://github.com/hashicorp/packer/blob/master/builder/amazon/instance/builder.go#L319)

Reply:
I added it to the other builders. :)